### ISSUE TITLE
Document multisig RPC calls

### ISSUE THREAD

**Issue by ghost** [monero-project/monero-site#522]
Multisig has now been merged into master via https://github.com/monero-project/monero/pull/2134

Any RPC calls that have been created to support this feature should be documented in the [wallet-rpc.md](https://github.com/monero-project/monero-site/blob/master/resources/developer-guides/wallet-rpc.md) file.

**Comment by el00ruobuob** [comment]
If @coneiric who's updated the subaddresses related calls in #694 has no time to do it, i will include this in a full update after #712 is finished and #694 is merged.

**Comment by erciccione** [comment]
@el00ruobuob was this issue resolved by #712 ?

**Comment by el00ruobuob** [comment]
No @erciccione there is more calls than those described along with subaddresses.
I still have to start refreshing the guide to write them down.

**Comment by el00ruobuob** [comment]
This will be solved in #854

**Comment by erciccione** [comment]
+in progress

**Comment by erciccione** [comment]
This issue is old and was discussed and closed on gitlab. Please reopen if you feel it's still relevant.
